src/Bellhop.js

```diff
@@ -262,31 +277,44 @@ export class Bellhop extends BellhopEventDispatcher {
   /**
    *  A convience method for listening to an event and then responding with some data
    *  right away. Automatically removes the listener
-   *  @method respond 
+   *  @method respond
    *  @param {String} event The name of the event
    *  @param {Object | function | Promise | string} [data = {}] The object to pass back.
    *  	May also be a function; the return value will be sent as data in this case.
    *  @param {Boolean} [runOnce=false] If we only want to respond once and then remove the listener
-   *  
+   *
    */
   respond(event, data = {}, runOnce = false) {
     const bellhop = this; //'this' for use inside async function
-    /** 
-     * @ignore 
-     */ 
-    const internalCallback = async function(event){
+    /**
+     * @ignore
+     */
+    const internalCallback = async function (event) {
       if (runOnce) {
         bellhop.off(event, internalCallback);
       }
-      if(typeof data === 'function'){
-        data = data(); 
+      if (typeof data === 'function') {
+        data = data();
       }
-      const newData = await data; 
+      const newData = await data;
       bellhop.send(event.type, newData);
     };
     this.on(event, internalCallback);
   }
 
+  /**
+   * Send either the default log message or the callback provided if debug
+   * is enabled
+   * @method logDebugMessage
+   */
+  logDebugMessage(received = false, message) {
+    if (this.debug && typeof this.debug === 'function') {
+      this.debug({ isChild: this.isChild, received: false, message: message });
+    } else if (this.debug) {
+      console.log(`Bellhop Instance (${this.isChild ? 'Child' : 'Parent'}) ${received ? 'Receieved' : 'Sent'}`, message);
```

Comment:
I was going to suggest making this a `console.debug` call instead, but it looks like [support is a tad patchy](https://caniuse.com/#feat=mdn-api_console_debug) so let's leave it as is 👍